src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -1463,60 +1614,141 @@ components:
+          items:
+            $ref: '#/components/schemas/ReasonNotToWork'
+        notHopingToWorkOtherReason:
           type: string
-          format: date-time
-          description: An ISO-8601 timestamp representing when the Induction was created.
-          example: '2023-06-19T09:39:44Z'
-        modifiedBy:
+          description: The reason that is given when the Prisoner does not want to work. This is mandatory when 'notHopingToWorkReasons' includes 'OTHER'.
+        affectAbilityToWork:
+          type: array
+          description: Factors affecting the Prisoner's ability to work. Can be empty but not null.
+          items:
+            $ref: '#/components/schemas/AbilityToWorkFactor'
+        affectAbilityToWorkOther:
           type: string
-          description: The DPS username of the person who last updated the Induction.
-          example: 'asmith_gen'
-        modifiedDateTime:
+          description: A specific factor affecting the Prisoner's ability to work. This is mandatory when 'affectAbilityToWork' includes 'OTHER'.
+      required:
+        - hopingToWork
+    CreatePreviousQualificationsRequest:
+      title: CreatePreviousQualificationsRequest
+      type: object
+      description: A request to create a Prisoner's qualifications.
+      properties:
+        educationLevel:
+          $ref: '#/components/schemas/HighestEducationLevel'
+        qualifications:
+          type: array
+          description: A list of the Prisoner's previous qualifications. Can be empty but not null.
+          items:
+            $ref: '#/components/schemas/AchievedQualification'
+    CreatePreviousTrainingRequest:
+      title: CreatePreviousTrainingRequest
+      type: object
+      description: A request to create a Prisoner's previous training.
+      properties:
+        trainingTypes:
+          type: array
+          description: Any additional training that the Prisoner has completed in the past. Must not be empty ('NONE' is an option).
+          items:
+            $ref: '#/components/schemas/TrainingType'
+        trainingTypeOther:
           type: string
-          format: date-time
-          description: An ISO-8601 timestamp representing when the Goal was last updated. This will be the same as the created date if it has not yet been updated.
-          example: '2023-06-19T09:39:44Z'
+          description: A specific type of training that does not fit the given 'trainingTypes'. Mandatory when 'trainingTypes' includes 'OTHER'.
       required:
-        - offenderId
-        - desireToWork
-        - hopingToGetWork
-        - createdBy
-        - createdDateTime
-        - modifiedBy
-        - modifiedDateTime
+        - trainingTypes
+    CreatePreviousWorkExperiencesRequest:
+      title: CreatePreviousWorkExperiencesRequest
+      type: object
+      description: A request to create a Prisoner's previous work experience.
+      properties:
+        hasWorkedBefore:
```

Comment:
It could be argued this is redundant based on the `experiences` array, but I think it's useful for convenience. Also, we know that the current screens do not send a list of work experiences when updating an Induction (because Gary didn't have clear requirements 😉), which is another (albeit lesser) reason to keep it.